tempto-core/src/main/java/com/teradata/tempto/query/QueryExecutor.java

```diff
@@ -56,6 +57,8 @@ QueryResult executeQuery(String sql, QueryType queryType, QueryParam... params)
 
     Connection getConnection();
 
+    Optional<String> getConnector();
```

Comment:
I don't like this method. Maybe it has some meaning in regards to presto connection but it cann't be mapped to anything in case of hive connection. I think that when you fix my other comment you will be able to remove this method.